_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -1867,10 +2572,132 @@ Finally, deploy the `account-baseline` module by running `terragrunt apply`:
 [source,bash]
 ----
 cd infrastructure-live/stage/_global/account-baseline
-terragrunt apply
+terragrunt apply -parallelism=4
+----
+
+IMPORTANT: We **strongly** recommend setting Terraform parallelism to a low value (e.g., `-parallelism=4`) with the `account-baseline-xxx` modules. This is because these modules deploy multi-region resources (e.g., GuardDuty, AWS Config, etc), and for each region, Terraform spins up a separate process, so if you don't limit the parallelism, it may peg all your CPU cores and lead to network connectivity errors.
```

Comment:
Is it worth repeating this recommendation so many times throughout the doc?

Reply:
I think so. As we said in the intro of the doc, skipping around is encouraged, so it's hard to say which location we do show this recommendation (if we were to have only one location).

Reply:
Yep, what Rho said! Including `-parallelism` is not common and I kept forgetting to do it myself while running this code, so I figured extra reminders would be useful here.